docs/adusting_with_depth_and_pressure.md

```diff
@@ -0,0 +1,155 @@
+(adjustment_section)=
+# Adjusting with Depth and Pressure
+
+The dry_rock model can be adjusted according to depth and pressure:
+
+```yaml
+dry_rock:
+    model:
+        # model type is one of:
+        #  * "polyfit"
+        #  * "friable_sand"
+        #  * "patchy_cement"
+        type: "polyfit"
+    porosity:
+        # porosity of dry rock gotten from column in csv file
+        column: "porosity"
+    # Default polyfit coefficients is used which is:
+    # density: [[0.0, 0.0], [1.0, -1.0]]
+    # bulk_modulus: [[2900.0, -1300.0]]
+    # shear_modulus: [[1700.0, -800.0]]
+    # In  dry_rock, effective reference pressure is used:
+    # pressure = min(max_pressure, overburden_pressure - reference_pressure)
+    adjustments:
+       - type: "pressure_dependency"
+         # In pressure dependency effective rock pressure is used:
+         # pressure = min(max_pressure, overburden_pressure - rock_pressure)
+         # With the exception of the powerfit model which uses rock pore pressure:
+         # pressure = rock_pressure
+         model:
+              # model type is one of:
+              # * "polyfit"
+              # * "expfit"
+              # * "logfit"
+              # * "powerfit"
+              # * "patchy_cement"
+              # * "friable_sand"
+             type: "powerfit"
+             coefficients:
+                # adjusted_bulk_modulus = original_bulk_modulus +
+                #         20.0 * delta_pressure^-40.0
+                bulk_modulus: [20.0E+9, -46.0]
+                # adjusted_shear_modulus = original_shear_modulus +
+                #         12.0 * delta_pressure^-20.0
+                vp_over_vs: [12.0, -26.0]
+                # adjusted_density = original_density
+                density: [1.0, 0.0]
+       - type: "depth_trend"
+         depth:
+           column: "depth"
+         reference_depth: 0.0
+         max_depth: 4000.0
+         # delta_depth = min(depth - reference_depth, max_depth)
+         # coefficients in a 2d polynomial which
+         # adjusts dry rock properties to depth
+         coefficients:
+           # adjusted_bulk_modulus = c[i][j] * delta_depth^i * original_bulk_modulus^j
+           bulk_modulus: [[0.0, 1.5], [1.0, 0.0]]
+           # adjusted_shear_modulus = c[i][j] * delta_depth^i * original_shear_modulus^j
+           shear_modulus: [[0.0, 1.2], [1.0, 0.0]]
+           # adjusted_density = c[i][j] * delta_depth^i * original_density^j
+           density: [[0.0, 1.35], [1.0, 0.0]]
+```
+
+In the above example, the dry_rock material changes both with depth and pressure according to
```

Comment:
Might be nice to use backticks for the names of things,eg `dry_rock`, `pressure_dependency`, etc. Personally I find this helps read docs with a lot of names floating about.